framer/LayerStates.coffee

```diff
@@ -15,6 +15,9 @@ deprecatedWarning = (name, suggestion) ->
 	message += ", use '#{suggestion}' instead." if suggestion?
 	console.warn message
 
+namedState = (state, name) ->
+	return _.extend({}, state, {name: name})
```

Comment:
Don't we want to enforce the named state by reversing `state` and `{name: name}`